testsuite/MDAnalysisTests/data/coordinates/test.xyz

```diff
@@ -6,28 +6,28 @@ frame 0
       CA     9.00000   10.00000   11.00000
       CA    12.00000   13.00000   14.00000
 5
-frame 0
+frame 1
```

Comment:
Please also create a new zipped file for the xyz format based on this. The readme says how this is done.